src/plugins/add-cal/add-cal-en.hbs

```diff
@@ -44,37 +46,54 @@
 <div class="wb-addcal provisional mrgn-bttm-md" typeof="Event">
 	<h3 property="name">Igloo Baratanga</h3>
 	<p property="description">A percussion activity for the whole family! Take part in a drum circle and get warmed up. No musical experience required and smiles guaranteed! Come play with us!</p>
-	<p><strong>When:</strong> From <time property="startDate" datetime="2020-02-08 13:00Z">Sunday, February 8, 2020 | 1:00 pm</time> to <time property="endDate" datetime="2020-02-08 13:30Z">1:30 pm</time></p>
-	<p property="location"><strong>Where:</strong> Snowflake Kingdom, <span lang="fr">Jacques-Cartier</span> Park, Gatineau</p>
+	<p><strong>When:</strong> From <time property="startDate" datetime="2023-02-08 13:00-04:00">Wednesday, February 8, 2023 | 1:00 pm</time> to <time property="endDate" datetime="2023-02-08 13:30-04:00">1:30 pm</time></p>
+	<p><strong>Where:</strong> <span property="location">Snowflake Kingdom, <span lang="fr">Jacques-Cartier</span> Park, Gatineau</span></p>
+</div>
+
+<h2>Example of an online event</h2>
+<div class="wb-addcal provisional mrgn-bttm-md" typeof="Event">
+	<h3 property="name">Marsai Martin’s Picks for a Social Summer</h3>
+	<p property="description">There’s no time like summer for trying new things and meeting new people, especially if you’ve been feeling lonely.</p>
+	<p><strong>When:</strong> From <time property="startDate" datetime="2023-08-08 13:00-04:00">Tuesday, August 8, 2023 | 1:00 pm</time> to <time property="endDate" datetime="2023-08-08 13:30-04:00">1:30 pm</time></p>
+	<p><strong>Where:</strong> <span property="location" typeof="VirtualLocation">Online (<a href="http://virtual.event" target="_blank" rel="noopener noreferrer" property="url">http://virtual.event</a>)</span></p>
```

Comment:
You can set the location property on the `<p>` instead of using a span.

```suggestion
	<p property="location" typeof="VirtualLocation"><strong>Where:</strong> Online (<a href="http://virtual.event" target="_blank" rel="noopener noreferrer" property="url">http://virtual.event</a>)</p>
```

Reply:
> You can set the location property on the `<p>` instead of using a span.

Doing that would add "Where:" at the beginning of the location on the generated event, i.e,: Location: Where: Online